### PR TITLE
Add assert to check warmup interval

### PR DESCRIPTION
**Pull request by nicmcd**
Solves one problem in #15.

### CODE REVIEW

**Comment by nicmcd** [comment]
@knrafto, I'm wondering if I should also add a minimum interval check in the constructor, such as 100 flits. Thoughts?

File: src/workload/blast/BlastTerminal.cc [hunk 1]
```diff
@@ -317,6 +317,7 @@ void BlastTerminal::warmDetector(Message* _message) {
   }
 
   // count flits received
+  assert(warmupInterval_ >= (2 * _message->numFlits()));
```
Comment (knrafto) [inline src/workload/blast/BlastTerminal.cc]: nit: redundant parens

Comment (nicmcd) [reply src/workload/blast/BlastTerminal.cc]: fixed

**Comment by nicmcd** [comment]
Ok, I added a minimum of 100. This makes sure each warm up sample contains at least 100 flits.

**Pull request merged**